The sphinxcontrib-issuetracker was disabled since it connects to various issue trackers via the internet and that won't work in the build environment.
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -33,7 +33,6 @@
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.coverage',
               'sphinx.ext.pngmath',
-              'sphinxcontrib.issuetracker',
               'celerydocs']
 
 # Add any paths that contain templates here, relative to this directory.
@@ -91,10 +90,3 @@
     '**': ['sidebarlogo.html', 'relations.html',
            'sourcelink.html', 'searchbox.html'],
 }
-
-### Issuetracker
-
-issuetracker = "github"
-issuetracker_user = "ask"
-issuetracker_project = "celery"
-issuetracker_issue_pattern = r'[Ii]ssue #(\d+)'
